cmd/dagger/cmd/lsp.go

```diff
@@ -11,7 +11,7 @@ import (
 )
 
 var lspCmd = &cobra.Command{
-	Use:    "lsp",
+	Use:    "cuelsp",
 	Short:  "Run Dagger CUE Language Server",
 	Hidden: true,
```

Comment:
Why do we hide it?

Reply:
It's gonna be easier to manage the future legacy of that subcommand